services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/events/BsmEvent/BsmEventRepositoryImpl.java

```diff
@@ -0,0 +1,116 @@
+package us.dot.its.jpo.ode.api.accessors.events.BsmEvent;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
suggestion: java.util.ArrayList doesn't appear to be used, let's remove

Reply:
removed